spec/runtime/setup_spec.rb

```diff
@@ -1318,22 +1317,18 @@ def lock_with(ruby_version = nil)
 
       it "activates no gems with -rbundler/setup" do
         install_gemfile! ""
-        ruby! code, :env => { :RUBYOPT => activation_warning_hack_rubyopt }
+        ruby! code, :env => { :RUBYOPT => activation_warning_hack_rubyopt + " -rbundler/setup" }
         expect(last_command.stdout).to eq("{}")
       end
 
       it "activates no gems with bundle exec" do
         install_gemfile! ""
-        # ensure we clean out the default gems, bceause bundler's allowed to be activated
         create_file("script.rb", code)
-        bundle! "exec ruby ./script.rb", :env => { :RUBYOPT => activation_warning_hack_rubyopt + " -rbundler/setup" }
+        bundle! "exec ruby ./script.rb", :env => { :RUBYOPT => activation_warning_hack_rubyopt }
```

Comment:
Why does this line no longer need `" -rbundler/setup"`?

Reply:
Because this spec is checking activation via `bundle exec` not via `-rbundler/setup` (as opposed to the previous spec, that uses `ruby` directly but passes `-rbundler/setup`).

Reply:
It's not that it no longer need it, it never actually needed it, as far as I can see.

Reply:
Thanks for the explanation!